R/unnest_tokens.R

```diff
@@ -154,10 +154,10 @@ unnest_tokens <- function(tbl, output, input, token = "words",
   output_lst <- tokenfunc(col, ...)
 
   if (!(is.list(output_lst) && length(output_lst) == nrow(tbl))) {
-    rlang::abort(
+    rlang::abort(sprintf(
       "Expected output of tokenizing function to be a list of length ",
```

Comment:
@juliasilge Sorry now that I look again at this it seems I forgot to insert a `%` placeholder in the message :(

Reply:
Yeah, I just noticed; I'll get that